index.html

```diff
@@ -185,15 +303,16 @@ <h1 class="greet">
           <li class="angelist"><a href="#"></a><img src="assets/Vector.png"></li>    
         </ul>
         <div class="resume">
-          <button class="button" id="resume">Get my resume</button>
+          <button type="button" class="button" id="resume">Get my resume</button>
         </div>
-
-        
+      </div>
+        <div class="grid_item2">
           <ul class="skills">
-              <ul class="languages">  
+              <ul class="languages"> 
                 <li><p>Languages</p></li>
-                <li><img src="assets/Unionlangs.png"></li>
+                <li class="arrow"><img src="assets/Unionlangs.png"></li>
```

Comment:
- [ ] Good job adding the drop down 👍. However, when you are adding it, please make sure that you follow the figma design according to the requirements. Right now the `down arrow` is misplaced. Please make sure that you align it horizontally and vertically exactly according to the figma template. I know you can do that 💪
- [OPTIONAL] Isn't it better to have a straight line for your right side of the column (marked in blue). Right now the right section is slightly misplaced. What do you think can we fix this? 🎯 I know you can do that 💪
## **_Your design previously 👇_**
![image](https://user-images.githubusercontent.com/106119105/213881513-455b07b2-2ddc-4fd4-b2c9-2ab434073db8.png)

## **_Your design now 👇_**
![image](https://user-images.githubusercontent.com/106119105/213881598-52b5cc91-c011-4153-b20e-28d653fe84d3.png)

## **_Figma design 👇_**
![image](https://user-images.githubusercontent.com/106119105/213881466-ac0179fc-b5d0-4f5f-b646-bd0e30809a1b.png)